src/farcasterd/runtime.rs

```diff
@@ -176,10 +176,10 @@ impl Runtime {
             return Err(Error::Syncer(SyncerError::InvalidConfig));
         }
         if !self.syncers.contains_key(&(coin_arb.clone(), network)) {
-            launch("syncerd", &[coin_arb.to_string(), network.to_string()])?;
+            launch("syncerd", &[coin_arb.to_string()])?;
         }
         if !self.syncers.contains_key(&(coin_acc.clone(), network)) {
-            launch("syncerd", &[coin_acc.to_string(), network.to_string()])?;
```

Comment:
if i were you, i would leave the stuff that is written to the toml to something runtime independent, such as a map btw say 
```
(Monero, Testnet) -> "--monero-daemon http://localhost:38081 --monero-rpc-wallet http://localhost:18083"
(Bitcoin, Testnet) -> "--electrum-server localhost:60001"
```
Then if the config doesnt have it, it can't run the swap.